### PR TITLE
Mxgxw/add scrapper integration

### PR DESCRIPTION
**Pull request by lekastillo**
Conectandose al scraping y enviado el Sms

### CODE REVIEW

**Review by trejo08 — APPROVED** [review]

File: config.ru [hunk 1]
```diff
@@ -1,3 +1,17 @@
 # config.ru
+require 'rack/env'
+require_relative './config/environment'
+
+if ENV['RACK_ENV']=='production'
+  redis_hash = { url: "redis://#{ENV['REDIS_HOST']}:#{ENV['REDIS_PORT']}", password: "#{ENV['REDIS_PASSWORD']}" }
+  use Rack::Env
```
Comment (trejo08) [inline config.ru:6]: this line should be before `redis_hash` because it's loading ENV vars

File: config.ru [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+else
+  redis_hash = { url: "redis://#{ENV['REDIS_HOST']}:#{ENV['REDIS_PORT']}" }
+  use Rack::Env, envfile: "#{File.expand_path(File.dirname(File.dirname(__FILE__)))}/.env"
```
Comment (trejo08) [inline config.ru:9]: this also should be before `redis_hash`

File: config.ru [hunk 3]
```diff
@@ -1,3 +1,17 @@
 # config.ru
+require 'rack/env'
+require_relative './config/environment'
+
+if ENV['RACK_ENV']=='production'
+  redis_hash = { url: "redis://#{ENV['REDIS_HOST']}:#{ENV['REDIS_PORT']}", password: "#{ENV['REDIS_PASSWORD']}" }
```
Comment (trejo08) [inline config.ru]: you should put this line after load `Rack::Env`

**Review by trejo08 — CHANGES_REQUESTED** [review]

File: config.ru (hunk 3 above)
Comment (eulesv) [reply config.ru]:
```suggestion
use Rack::Env
  redis_hash = { url: "redis://#{ENV['REDIS_HOST']}:#{ENV['REDIS_PORT']}", password: "#{ENV['REDIS_PASSWORD']}" }
```

**Review by trejo08 — APPROVED** [review]

**Pull request merged**